apps/browser/supabase/migrations/20231206012608_chat_history.sql

```diff
@@ -0,0 +1,83 @@
+drop table roles CASCADE;
+drop table chats CASCADE;
+drop table messages CASCADE;
+
+CREATE TABLE "public"."chats" (
+  "id" "uuid" DEFAULT "extensions"."uuid_generate_v4"() NOT NULL,
+  "created_at" timestamp with time zone DEFAULT "now"() NOT NULL,
+  "user_id" "uuid" default auth.uid(),
+  PRIMARY KEY ("id"),
+  FOREIGN KEY ("user_id") REFERENCES "public"."users"("id")
+);
+
+alter table "public"."chats" enable row level security;
+
+create table "public"."logs" (
+  "id" "uuid" DEFAULT "extensions"."uuid_generate_v4"() NOT NULL,
+  "created_at" timestamp with time zone not null default now(),
+  "title" text not null,
+  "content" text,
+  "chat_id" uuid not null,
+  "user" text not null,
+  FOREIGN KEY ("chat_id") REFERENCES "public"."chats"("id")
+);
+
+alter table "public"."logs" enable row level security;
+
+create table "public"."variables" (
+  "id" "uuid" DEFAULT "extensions"."uuid_generate_v4"() NOT NULL,
+  "key" text not null,
+  "value" text not null,
+  "chat_id" uuid not null,
+  FOREIGN KEY ("chat_id") REFERENCES "public"."chats"("id")
+);
+
+alter table "public"."variables" enable row level security;
+
+CREATE TABLE "public"."messages" (
+  "id" "uuid" DEFAULT "extensions"."uuid_generate_v4"() NOT NULL,
+  "created_at" timestamp with time zone DEFAULT "now"() NOT NULL,
+  "chat_id" uuid NOT NULL,
+  "content" "text",
+  "name" "text",
+  "function_call" "json",
+  "temporary" boolean NOT NULL,
+  "tool_call_id" "text",
+  "tool_calls" json,
+  "role" "text" NOT NULL,
+  PRIMARY KEY ("id"),
+  FOREIGN KEY ("chat_id") REFERENCES "public"."chats"("id")
+);
+
+CREATE UNIQUE INDEX logs_pkey ON public.logs USING btree (id);
+
+CREATE UNIQUE INDEX variables_pkey ON public.variables USING btree (id);
+
+alter table "public"."logs" add constraint "logs_pkey" PRIMARY KEY using index "logs_pkey";
+
+alter table "public"."variables" add constraint "variables_pkey" PRIMARY KEY using index "variables_pkey";
+
+alter table "public"."logs" validate constraint "logs_chat_id_fkey";
```

Comment:
`"logs_chat_id_fkey"` Is this a typo? There is no constraint named `"logs_chat_id_fkey"` previously defined. Is this by chance pre-defined whenever you create a new `FOREIGN KEY`?

Reply:
`logs_chat_id_fkey` is created when creating the foreign key. 

I will remove this since the objective of this query is to make sure that the log table doesn't have an orphan row (we were modifying it with a query given by the `diff` command from supabase); and this is not the case anymore